dev-requirements.txt

```diff
@@ -5,10 +5,16 @@ flake8~=5.0
 Flake8-pyproject~=1.2
 isort~=5.11
 moto~=4.1.10
+mypy-boto3-athena~=1.26
+mypy-boto3-glue~=1.26
+mypy-boto3-lakeformation~=1.26
 pre-commit~=2.21
+pyathena~=2.25
```

Comment:
You should not have to add the prod requirements in the dev requirements file.
You can use `pip install -e .` for that

Reply:
ah right, I just assumed `dev-dependencies` would contain all dependencies, but checking the Makefile it's clear.
Still, maybe something like `poetry` or `pipenv` would be nice to have.

will revert the change to the `dev-requirements.txt` file

Reply:
We discussed about `poetry` and `pipenv` but right now no adapter is using it so we went for the same approach. If dependencies handling is hard, we might reconsider this decision 🙂